go.mod

```diff
@@ -1,6 +1,6 @@
 module decred.org/dcrdex
 
-go 1.15
+go 1.16
 
 replace decred.org/dcrdex/internal/eth/reentryattack => ./internal/eth/reentryattack
```

Comment:
I guess you can take this out now if you want. I think we just needed it before the new package was merged to master.

Reply:
Yeah, removed.  Strange though because "decred.org/dcrdex/internal/eth/reentryattack" isn't even a module.